### DISCUSSION TITLE
Nix-env install not working - is it due to static build problem?

### DISCUSSION THREAD

**Discussion by thatdwayne** [srid/neuron#668] [Installation]
MacOS - running "nix-env -if https://github.com/srid/neuron/archive/master.tar.gz" - no error, just an immediate, successful exit with nothing installing. Is this due to the static builds issue?

Anyone know of an alternative? I can download the master.tar.gz file - is there a way to install from a local copy with nix? I don't know nix well enough to be able to figure it out. Any guidance much appreciated.

**Comment by srid** [comment]
This is nothing to do with static builds. Probably an issue with your Nix install. 

> is there a way to install from a local copy with nix? 

You can run `nix-env -if .` from inside the source directory.
